server/handlers/handlers.go

```diff
@@ -295,6 +290,13 @@ func (ah *ActivityHandler) handleCreatedActivity(activityIds msteams.ActivityIds
 	}
 
 	ah.plugin.GetAPI().LogDebug("Post created", "post", newPost)
+	if errorsFound {
+		_ = ah.plugin.GetAPI().SendEphemeralPost(senderID, &model.Post{
```

Comment:
whats the return type of this and if error do we need to test it.

Reply:
Its return type is `model.Post`.